### PR TITLE
Feature/hel 1972

### PR DESCRIPTION
**Pull request by rajeshmohanty-leanix**
Initial PR for sending status (operational) event for k8s scan

### CODE REVIEW

File: pkg/iris/events.go [hunk 1]
```diff
@@ -75,3 +76,43 @@ type Log struct {
...
+func NewStatusEvent(configurationId string, runId string, workspaceId string, runstatus string, message string) *StatusItem {
+
+	id := fmt.Sprintf("%s", runId)
+	Subject := fmt.Sprintf("Status")
+	Type := fmt.Sprintf("leanix.vsm.item-logged.status")
+	scope := fmt.Sprintf("workspace/%s", workspaceId)
+	Source := fmt.Sprintf("kubernetes/%s#%s", configurationId, runId)
+	Time := fmt.Sprintf(time.Now().Local().String())
+	datacontenttype := fmt.Sprintf("application/json")
+	dataschema := fmt.Sprintf("/vsm-iris/schemas/operation-item/v1")
```
Comment (DennisBerthold-leanix) [inline pkg/iris/events.go:101]: some are lower case, some upper case. let's consolidate on lower case and use camelCase

Comment (rajeshmohanty-leanix) [reply pkg/iris/events.go]: Updated to camelCase.

Comment (rajeshmohanty-leanix) [reply pkg/iris/events.go]: But we have to keep the json attributes all lowercase as expected.

**Review by daniellesarmiento — APPROVED** [review]

File: pkg/iris/api.go [hunk 2]
```diff
@@ -70,3 +71,30 @@ func (a *api) PostResults(results []byte, accessToken string) error {
 	log.Infof("Event posted successfully [%s]", resp.Status)
 	return nil
 }
+
+func (a *api) PostStatus(status []byte, accessToken string) error {
```
Comment (SudharmaPuranikLeanIX) [inline pkg/iris/api.go:75]: Just for understanding. Does post status should be different from post results? I understood post result should be generic to use.

File: pkg/iris/api.go [hunk 3]
```diff
@@ -70,3 +71,30 @@ func (a *api) PostResults(results []byte, accessToken string) error {
...
+	resultUrl := fmt.Sprintf("https://%s/services/vsm-iris/v1/status", a.uri)
```
Comment (SudharmaPuranikLeanIX) [inline pkg/iris/api.go:76]: what is the difference between `/status` & `/results`

**Review by SudharmaPuranikLeanIX — CHANGES_REQUESTED** [review]
Does post status should be different from post results? I understood post result should be generic to use.

File: pkg/iris/api.go (hunk 3 above)
Comment (rajeshmohanty-leanix) [reply pkg/iris/api.go:76]: Endpoints `/status` and `/results` have different schema objects and the respective target eventbus are also different such as 'discovery_items' and 'feedback_items'.

**Review by SudharmaPuranikLeanIX — APPROVED** [review]
🚀

**Review by daniellesarmiento — APPROVED** [review]

**Pull request merged**